[PRP-347] Participant Documentation (#114)
navapbc/wic-participant-recertification-portal: docs/participant/pages/09_confirm.md

# Confirm

### About this page:

This page confirms the submission and shows content about next steps. It contains an optional user survey for feedback about their experience. It shows the user's submitted information.

### Considerations for states:

- You may want to update next steps content with agency specific next steps.

> - You can replace your own survey link with your own to collect user feedback.

Reviewer: ```suggestion
- Replace the survey link with your own to collect user feedback.
```

### Development

> The Confirm page renders all previous page data stored in `SubmissionForm` (and a count of `Document` records) in
> a special component (not in Storybook) called a [`<SubmissionForm>`](../../../participant/app/components/SubmissionForm.tsx)
> 
> The applicant cannot edit this page, and navigation will ask the individual if they would like to reset their cookie
> to allow completing the form a second (or nth) time.

Reviewer: ```suggestion
The Confirm page renders all previous page data stored in `SubmissionForm` (and a count of `Document` records) in a special component (not in Storybook) called a [`<SubmissionForm>`](../../../participant/app/components/SubmissionForm.tsx)

Once the participant reaches this page, they are not longer able to edit any information. If they click the back button in their browser, an alert will appear asking the individual if they would like to reset their cookie to allow completing the form a second (or nth) time.
```
